### DISCUSSION TITLE
How i can get the text from the textarea

### DISCUSSION THREAD

**Discussion by refaatcrafts** [charmbracelet/bubbletea#912] [Q&A]
Hello guys,
i was wandering how i can get he text for the textare and how i can change the color of the text in real time 

i want to compere between  the two texts in the two textare and find the difference words and  marked or change the color of it in real time
![image](https://github.com/charmbracelet/bubbletea/assets/76957887/79920829-5794-4166-856f-399508976ff3)

**Comment by maaslalani** [comment]
Hi @nerdxio, you might be looking for the `Value()` method!

https://github.com/charmbracelet/bubbles/blob/v0.17.1/textarea/textarea.go#L399
